lib/agent/socket_util.js

```diff
@@ -11,7 +11,7 @@ const OcspResponseCache = require('./ocsp_response_cache');
 const Errors = require('../errors');
 const ErrorCodes = Errors.codes;
 
-const REGEX_SNOWFLAKE_ENDPOINT = /.snowflakecomputing.com$/;
+const REGEX_SNOWFLAKE_ENDPOINT = /.snowflakecomputing.(com|cn)?$/;
```

Comment:
it's not correct when there will be a different top level domain - may we just leave `/.snowflakecomputing./`?